framework/widgets/Pjax.php

```diff
@@ -168,7 +168,6 @@ public function run()
         $response->format = Response::FORMAT_HTML;
         $response->content = $content;
         $response->headers->setDefault('X-Pjax-Url', Yii::$app->request->url);
-        $response->send();
```

Comment:
Is it still sending `X-Pjax-Url` header and the content?

Reply:
Yes it. `Application#end()` automatically trigger events and send response.

https://github.com/yiisoft/yii2/blob/ac8307d5aa7c657cf71c2a0df243c1812eb6e59b/framework/base/Application.php#L656

https://github.com/yiisoft/yii2/blob/ac8307d5aa7c657cf71c2a0df243c1812eb6e59b/framework/base/Application.php#L639-L664